hotshot-task-impls/src/upgrade.rs

```diff
@@ -160,6 +160,46 @@ impl<TYPES: NodeType, V: Versions> UpgradeTaskState<TYPES, V> {
                     proposal.data.view_number()
                 );
 
+                let epoch_upgrade_checks = if V::Upgrade::VERSION == V::Epochs::VERSION {
```

Comment:
Just to confirm this is and should be a copy of the same checks on the proposal side.  Looks the same to me but just wanted a double check